lib/report_plugins/treatments.js

```diff
@@ -237,13 +265,16 @@ treatments.report = function report_treatments(datastorage, sorteddaystoshow, op
     .append($('<th>').css('width','150px').attr('align','left').append(translate('Blood Glucose')))
     .append($('<th>').css('width','50px').attr('align','left').append(translate('Insulin')))
     .append($('<th>').css('width','50px').attr('align','left').append(translate('Carbs')))
+    .append($('<th>').css('width','50px').attr('align','left').append(translate('Duration')))
+    .append($('<th>').css('width','50px').attr('align','left').append(translate('Percent')))
+    .append($('<th>').css('width','50px').attr('align','left').append(translate('Basal value')))
     .append($('<th>').css('width','150px').attr('align','left').append(translate('Entered By')))
     .append($('<th>').css('width','300px').attr('align','left').append(translate('Notes')))
   );
   
   sorteddaystoshow.forEach(function (day) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1158641968/issues/source?bid=2452374&fileBranchId=2915302#l275)